indico/MaKaC/webinterface/tpls/RoomBookingNewBookingCollaborationWidget.tpl

```diff
@@ -18,7 +24,11 @@
     </div>
     <ul id="videoconferenceOptions" class="weak-hidden">
         % for subfield in [form.needs_vc_assistance] + eq_list:
-            <li class="js-vc-row">
+            % if subfield.short_name == 'needs_vc_assistance' and start_time < work_start or start_time > work_end:
+                <li class="js-vc-row unavailable" title="${ _('Assistance is not available because start time of booking is not within working hours.') }">
+            % else:
+                <li class="js-vc-row">
+            % endif
                 ${ subfield() }
```

Comment:
It doesn't work I get : NameError: Undefined , neither does : `disabled='true'` , although `readonly='true'` works, but thats not really what we want ..